device/vxlan/destinationipmode.yaml

```diff
@@ -1,16 +1,31 @@
 components:
   schemas:
-    DestinationIPMode:
+    Vxlan.V4Tunnel.DestinationIPMode:
       description: >-
-        Communication Mode between the VMs, choice as either multicast or unicast
```

Comment:
Communication mode between the vteps,choice is either multicast or unicast.

Reply:
updated